Modules/DevTools/TestFramework/TestLibraries/Any/src/Any.Codeunit.al

```diff
@@ -44,8 +44,17 @@ codeunit 130500 "Any"
     /// <param name="DecimalPlaces">Number of decimal places</param>
     /// <returns>Pseudo-random decimal value</returns>
     procedure DecimalInRange(MaxValue: Integer; DecimalPlaces: Integer): Decimal
+    var
+        x: Integer;
+        Pow: Integer;
```

Comment:
I would rename Pow to NumberWithLengthEqualsDecimalPlaces. This is much more understandable for future code readers

Reply:
If you decide to rename Pow here, you'll need to update it also in 

- `procedure DecimalInRange(MinValue: Decimal; MaxValue: Decimal; DecimalPlaces: Integer): Decimal`
- `procedure DecimalInRange(MaxValue: Integer; DecimalPlaces: Integer): Decimal`

Reply:
It is too long, Pow or Power is intuitive enough. Let's keep the change small.

Reply:
I would use the very descriptive name if the method longer, for very a small method like this one, Pow or Power will do.